extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/jdbc/DBDelegateUtils.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.quartz.runtime.jdbc;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ObjectInputStream;
+import java.sql.Blob;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+class DBDelegateUtils {
+
+    static Object stdGetObjectFromBlob(ResultSet rs, String colName) throws ClassNotFoundException, IOException, SQLException {
```

Comment:
Could you add some javadoc to explain why some delegates use this method and others the `getObjectFromInput()` method?

Reply:
Good idea thanks. I'll add the javadoc to the `getObjectFromInput()` method, and the rest of the delegate classes.

Reply:
done